.test-infra/jenkins/job_beam_PostCommit_Java_Nexmark_Direct.groovy

```diff
@@ -0,0 +1,80 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import common_job_properties
+
+// This job runs the suite of ValidatesRunner tests against the Direct runner.
+job('beam_PostCommit_Java_Nexmark_Direct') {
+  description('Runs the Nexmark suite on the Direct runner.')
+
+  // Set common parameters.
+  common_job_properties.setTopLevelMainJobProperties(delegate, 'master', 240)
+
+  // Sets that this is a PostCommit job.
+  common_job_properties.setPostCommit(delegate)
+
+  // Allows triggering this build against pull requests.
+  common_job_properties.enablePhraseTriggeringFromPullRequest(
```

Comment:
On dev@ there were a bunch of discussions of not doing phrase triggering on postcommits, but making a separate job. Is that ready to be used? Not blocking, as long as this job matching the status quo.

Reply:
This ones are more validates runners jobs. I think being able to trigger them from the PRs are a good way to ensure that you don't break anything when modifying a given runner.

Reply:
+1 cf my comment bellow